views/partials/email_partials/report.hbs

```diff
@@ -1,67 +1,72 @@
-{{!< default_email}}
-
-  <!--report subhead-->
-  <table width="100%">
-    <tr>
-      <td align="center" style="max-width: 450px;">
-        <p class="email-subhead" style="color: rgb(32, 18, 58); font-size: 16px; font-weight: 400; font-family: sans-serif; text-align: center; margin: 0; padding: 0 0 20px 0; line-height: 1.5;">
-          {{#if breachAlert}}
-            {{ getString "email-breach-alert-blurb" }}
-          {{else}}
-            {{#ifCompare unsafeBreachesForEmail.length "===" 0}}
-              {{ getString "fxm-warns-you-no-breaches" }}
-            {{else}}
-              {{ getString "fxm-warns-you-found-breaches" }}
-            {{/ifCompare}}
-          {{/if}}
-        </p>
-      </td>
-    </tr>
-  </table>
-  
-  {{#if breachAlert}}
-    <!--breach alert -->
-    <table width="100%" style="max-width: 460px;">
+<tr>
+  <td style="color: black; background: #F9F9FA; padding: 36px 0 24px;">
+    <p>
+      {{#ifCompare unsafeBreachesForEmail.length "===" 0}}
+      {{ getString "fxm-warns-you-no-breaches" }}
+      {{else}}
+      {{{ getString "email-breach-detected" email-address=( concat "<strong>" breachedEmail "</strong>" ) }}}
```

Comment:
Can we chat a bit more about this?

Reply:
We had a chat about this. The potential issue is not the demo email page, which is locked down to admins. Instead, I'm concerned about when this same report is sent to end users using an email that they entered into the site, since we don't do any HTML escaping of the `breachedEmail`.

We have some protections, such as email validation, to prevent invalid data getting into the database in the first place, so we have quite a few defenses. Additionally, an attacker would need to construct a string that is also a valid email, or the email won't be delivered - I wasn't able to find an example with a quick search.

Handlebars allows for [subexpressions](https://handlebarsjs.com/guide/expressions.html#subexpressions), and has an [escapeExpression](https://handlebarsjs.com/api-reference/utilities.html#handlebars-escapeexpression-string) helper, so this _might_ work:

```suggestion
      {{{ getString "email-breach-detected" email-address=( concat "<strong>" ( escapeExpression breachedEmail ) "</strong>" ) }}}
```

@toufali says that our implementation of handlebars doesn't include the full toolbox by default, and it may be a pain to add it, but he'll investigate.